src/tools/dart-run.md

```diff
@@ -47,17 +54,13 @@ in the pubspec, specify the package name.
 For example, to run `bar.dart` in the foo package:
```

Comment:
I would rephrase this as something like: runs `bin/bar.dart` from `package:foo`.

Reply:
TODO: we should also highlight the shorthand `dart run foo` is equivalent to `dart run foo:foo`, which means it runs `bin/foo.dart` from `package:foo`.